javascript/javascript3/week3/test.js

```diff
@@ -0,0 +1,44 @@
+//Lets use the github api to see what repositories different users have.
+
+const repositoryUl = document.getElementById("repositoryUl");
```

Comment:
I assume test.js shouldn't be here as it contains almost the same code as promises.js. Will just ignore. But it's confusing.